### PR TITLE
RDCC-3923: Remove Deprecated Jcenter Repository

### PR DESCRIPTION
**Pull request by ishratt**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-3923

### Change description ###
Removed deprecated jcenter repository

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: .idea/dbnavigator.xml [hunk 1]
```diff
@@ -0,0 +1,458 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shreedhar-hmcts) [inline .idea/dbnavigator.xml]: are .idea/dbnavigator.xml and .idea/misc.xml are required ?

Comment (ishratt) [reply .idea/dbnavigator.xml]: no sorry, they got committed by mistake, will delete them. Thanks @shreedhar-hmcts

**Review by SujeethKumarAri — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -287,10 +287,11 @@ dependencies {
 
     compile group: 'com.sun.xml.bind', name: 'jaxb-osgi', version: '2.3.3'
 
-    compile group: 'uk.gov.hmcts.reform', name: 'logging', version: versions.reformLogging
-    compile group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
-    compile group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version: '0.1.0'
-    compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: versions.reformS2sClient
+    implementation 'com.github.hmcts.java-logging:logging:5.1.7'
```
Comment (SujeethKumarAri) [inline build.gradle]: can you upgrade versions.reformLogging to latest and use that instead of using hard code `5.1.7`

File: build.gradle [hunk 3]
```diff
@@ -287,10 +287,11 @@ dependencies {
...
+    implementation 'com.github.hmcts.java-logging:logging-appinsights:5.1.7'
```
Comment (SujeethKumarAri) [inline build.gradle]: can you upgrade versions.reformLogging to latest and use that instead of using hard code `5.1.7`

**Review by SujeethKumarAri — CHANGES_REQUESTED** [review]

**Review by SujeethKumarAri — APPROVED** [review]

**Review by shreedhar-hmcts — APPROVED** [review]

**Pull request merged**